### DISCUSSION TITLE
ROSETTA NOT RUNNING. "See LICENSE.md or email license@uw.edu for more details" popping up.

### DISCUSSION THREAD

**Discussion by intomybioverse** [RosettaCommons/rosetta#85] [Rosetta]
Hello. 
RosettaDNA application is not running! 
I am using it exclusively for academic purpose. I have installed the rosetta academic package form https://rosettacommons.org/software/academic. I found the LICENSE.md file at https://github.com/RosettaCommons/rosetta/blob/main/LICENSE.md, but did not find any form to submit for the academic license.  
This is what I am getting.

$ /home/soumya/Downloads/rosetta.source.release-371/main/source/bin/rosettaDNA.linuxgccrelease @design_bg7gua1.flags
********  (C) Copyright Rosetta Commons Member Institutions.  ***************
* Use of Rosetta for commercial purposes may require purchase of a license. *
********  See LICENSE.md or email license@uw.edu for more details. **********
apps.public.rosettaDNA: *********************successful completion**************************

Please help out.

**Comment by roccomoretti** [comment]
You don't need a license or a license file for non-commercial, academic use. The message that you see is the copyright message that's always printed out for Rosetta runs, with a reminder that if you are using it for commercial purposes (e.g. consulting work) you would need to get a separate license.

It looks like you are getting the message "apps.public.rosettaDNA: successful completion", which indicates the program is running and at least thinks it's successfully completed. Have you checked for output files? -- I'll also note that most Rosetta protocols have "restart" behavior, in that they'll look to see if the output files that they would have generated already exist, and if they do, it won't bother to re-run those outputs. If you already have the nominal output files from a different run, Rosetta may just be skipping the output.

I will say that the amount of tracer output you're getting is rather small. My guess is that your design_bg7gua1.flags file contains `-mute` or `-out:levels` lines, which is turning off output for certain tracers. I would recommend commenting out any such lines while you're working out the protocol, and then only re-enable them when things are working well and you're doing production runs.

**Comment by roccomoretti** [comment]
Marking this as "closed", as I hope your question has been answered. If that assessment is incorrect, feel free to re-open or open up a new discussion with more details.
